bundles/io/org.eclipse.smarthome.io.rest.auth/src/main/java/org/eclipse/smarthome/io/rest/auth/AbstractSecurityHandler.java

```diff
@@ -27,10 +27,14 @@
 /**
  * Base type for handling authentication and authorization delegated from rest/jaxrs service layer.
  *
+ * @deprecated Since introduction of security on lower layer (servlets), this extension which is
```

Comment:
I'd be ok with completely removing this class - I doubt that anyone is using this as it never fully worked.